python/GafferSceneUI/SceneViewUI.py

```diff
@@ -51,15 +51,9 @@
 
 	GafferSceneUI.SceneView,
 
-	# We want our EditScopePlugValueWidget and _StateWidget to be grouped on the
-	# right, and we want our other settings to be grouped in the centre. To achieve
```

Comment:
We've lost the centering of the central group of widgets.

Reply:
Good spot thanks, fixed [here](https://github.com/GafferHQ/gaffer/pull/3832/commits/0302f48b942f42cbf71f2a3a7a6a0371d2b18698) and [here](https://github.com/GafferHQ/gaffer/pull/3832/commits/48817f2f32813ff586824f121d638094a04f8e97).